src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Resources/SR.resx

```diff
@@ -144,6 +144,9 @@
   <data name="Language_Services_Missing_Service" xml:space="preserve">
     <value>Razor language services not configured properly, missing language service '{0}'.</value>
   </data>
+  <data name="Not_Available_In" xml:space="preserve">
+    <value>Not available in</value>
```

Comment:
```suggestion
    <value>Not available in {0}</value>
```

IIRC we have to do it like this because placement can change depending on the language

Reply:
I'm deliberately laying the availability out over multiple lines, so I don't think any other placement would work. This is essentially a heading, not part of a sentence